routes/teams.js

```diff
@@ -31,17 +31,16 @@ router.get('/', function(res, res) {
 router.get('/quiz', function(req, res) {
   var rb_team_id = req.query.rb_team_id;
   var league = req.query.league;
-  var list_id = req.query.list_id;
   var type = req.query.type;
 
   if (req.query.trending){
-    if (type == 'leaders'){
-      db.collection('leaders').findOne( { team_id : rb_team_id}, function (err, item){
```

Comment:
yea- we had been querying the leaders collection twice. now we do teams once leaders once